index.html

```diff
@@ -62,8 +62,203 @@ <h1>I’m Taflon Glad to see you!</h1>
+                       <a href="#">css</a>
+                       <a href="#">Javascript</a>
+                    </div>
+                    <div class="last-button">
+                       <button>See Project</button>
+                    </div>
+                </div>
+
+                <div class="card_one">
+                    <div class="card_image">
+                       <img src="images/Snapshoot Portfolio_4.png" alt="">
+                    </div>
+                    
+                    <h2>Multi-Post
+                        Stories</h2>
+                    <div class="card_details">
+                       <ul>
+                           <li class="canopy">CANOPY</li>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                       </ul>
+                      
+                       <p>
+                           A daily selection of privately personalized reads; no accounts or sign-ups required.
+                       </p>
+                    </div>
+                    <div  class="first_buttons">
+                       <a href="#">html</a>
+                       <a href="#">css</a>
+                       <a href="#">Javascript</a>
+                    </div>
+                    <div class="last-button">
+                       <button>See Project</button>
+                    </div>
+                </div>
+
+               
+               
+            </div>
+          
+        </section>
+        <section class="about_page">
+            <div class="main_container">
+               <h2 class="about_myself_header"> About Myself</h2> 
+               <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+
+               </p>
+               <div class="connect">
+                LET’S CONNECT
+               </div>
+               <ul>
+                 <li>
+                    <a href="#">
+                        <img src="images/Vector-twiter.png" alt="">
+                    </a>
+                 </li>
+                 <li><a href="#">
+                     <img src="images/Linkedin icon.png" alt="">
+                   </a>
+                 </li>
+                 <li>
+                    <a href="#">
+                        <img src="images/Vector_M.png" alt="">
+                    </a>
+                 </li>
+                 <li>
+                    <a href="#">
+                        <img src="images/Vector_peace.png" alt="">
+                    </a>
+                 </li>
+                 <li>
+                    <a href="#"><img src="images/Vector_dog.png" alt="">
+                    </a>
+                 </li>
+               </ul>
+               <div class="about-button">
+                <button>Get my resume</button>
+             </div>
```

Comment:
- [ ] Dear @Tafloninno, for your `Get my resume` buttons we need to follow the button styling: `Active`, `Hover `and `Pressed`
![image](https://user-images.githubusercontent.com/69077061/212305601-6f2d27a2-1c05-4aed-bdd2-17351d0f8f57.png)